.github/workflows/build.yml

```diff
@@ -20,11 +20,14 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: ^1.22
```

Comment:
Not related to your change but related to my other comment

https://github.com/cosmtrek/air/pull/592#discussion_r1617032972

Here you can use "stable" it will always use the latest version and you don't have to maintain it

Reply:
We could but it has downsides as well. It may be "easier" to maintain but may break all the users of the project, which has a bigger impact than just change the version in every few months. See https://vsupalov.com/docker-latest-tag/

Reply:
> We could but it has downsides as well. It may be "easier" to maintain but may break all the users of the project, which has a bigger impact than just change the version in every few months. See https://vsupalov.com/docker-latest-tag/

Very good read, I get it.